ios/FluentUI/Notification/NotificationView.swift

```diff
@@ -505,40 +413,43 @@ open class NotificationView: UIView {
             hasSingleLineLayout = (messagelabelSize.height == messageLabel.font.deviceLineHeight)
         } else {
             let titleLabelSize = titleLabel.systemLayoutSizeFitting(CGSize(width: availableLabelWidth, height: 0), withHorizontalFittingPriority: .defaultHigh, verticalFittingPriority: .defaultHigh)
-            suggesgedHeight += titleLabelSize.height
+            suggestedHeight += titleLabelSize.height
         }
 
-        let suggestedVerticalPadding = hasSingleLineLayout ? Constants.verticalPaddingForOneLine : Constants.verticalPadding
-        suggesgedHeight += 2 * suggestedVerticalPadding
-        suggesgedHeight = ceil(max(suggesgedHeight, hasSingleLineLayout ? Constants.minHeightForOneLine : Constants.minHeight))
+        let suggestedVerticalPadding: CGFloat! = hasSingleLineLayout ? tokens.verticalPaddingForOneLine : tokens.verticalPadding
+        suggestedHeight += 2 * suggestedVerticalPadding
+        suggestedHeight = ceil(max(suggestedHeight, hasSingleLineLayout ? tokens.minimumHeightForOneLine : tokens.minimumHeight))
         container.layoutMargins = UIEdgeInsets(top: suggestedVerticalPadding, left: 0, bottom: suggestedVerticalPadding, right: 0)
 
-        return CGSize(width: suggestedWidth, height: suggesgedHeight)
+        return CGSize(width: suggestedWidth, height: suggestedHeight)
     }
 
     open override var intrinsicContentSize: CGSize {
         return sizeThatFits(CGSize(width: CGFloat.greatestFiniteMagnitude, height: CGFloat.greatestFiniteMagnitude))
     }
 
     private func updateForStyle() {
-        clipsToBounds = !style.needsSeparator
-        layer.cornerRadius = style.cornerRadius
+        clipsToBounds = !state.style.needsSeparator
+        layer.cornerRadius = tokens.cornerRadius
         layer.cornerCurve = .continuous
+        layer.masksToBounds = false
+        layer.shadowColor = tokens.shadowColor
+        layer.shadowRadius = tokens.shadowBlur
+        layer.shadowOffset = CGSize(width: tokens.shadowOffsetX, height: tokens.shadowOffsetY)
+        layer.shadowOpacity = 1.0
 
-        separator.isHidden = !style.needsSeparator
+        separator.isHidden = !state.style.needsSeparator
 
         updateWindowSpecificColors()
     }
 
     private func updateWindowSpecificColors() {
-        if let window = window {
-            backgroundView.updateBackground(backgroundColor: style.backgroundColor(for: window))
-            let foregroundColor = style.foregroundColor(for: window)
-            imageView.tintColor = foregroundColor
-            titleLabel.textColor = foregroundColor
-            messageLabel.textColor = foregroundColor
-            actionButton.tintColor = foregroundColor
-        }
+        self.backgroundColor = tokens.backgroundColor
```

Comment:
I don't see any code which you are letting the tokens know that color proxy might have changed?